### ISSUE TITLE
Were is the Workflow

### ISSUE THREAD

**Issue by kakachiex2** [ZHO-ZHO-ZHO/ComfyUI-SVD-ZHO#1]
Workflow please and there is a node that have camera movement option maybe can be good to did a check

https://github.com/alt-key-project/comfyui-dream-video-batches

**Comment by lunatico67** [comment]
Yes, it would be wonderful to have workflows to be able to use it.

**Comment by rodsott** [comment]
Yes, I didn't get it either, Zho. Please show how to use your 3 nodes, at your coffee video example, we dont see nothing, at least if we have the workflow even in a jpeg to reproduce, would be useful.. Thanks in advance!!!
